<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle de cart√µes</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame2{
background-color:#c8cbcb;
border-radius:8px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ETS - ADM</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAdm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>R$ 1.200,30</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Montagem das faturas por centro de custos</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame2_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame2_2{
background-color:#c8cbcb;
border-radius:8px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ETS - OFICINA</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSomaOficina">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>R$ 2.200,30</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame2_3{
background-color:#c8cbcb;
border-radius:8px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ETS - ALPHA</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSomaAlpha">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>R$ 2.200,30</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame2_4{
background-color:#c8cbcb;
border-radius:8px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ETS - MAQ E EQUIP</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMaquinas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>R$ 2.200,30</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame2_5{
background-color:#c8cbcb;
border-radius:8px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ETS - JAIBA LOC</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblJaiba">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>R$ 2.200,30</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>721</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame2_6{
background-color:#c8cbcb;
heigth:3500px;
width:300px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>6</number>
    </property>
    <widget class="QLabel" name="btnListFatura">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>list_alt_24dp_5F6368_FILL0_wght400_GRAD0_opsz24.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAbriFatura">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Listar fatura</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>65</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>add_card_24dp_5F6368_FILL0_wght400_GRAD0_opsz24.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddCartao">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Add - Cartao</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>129</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>person_add_24dp_5F6368_FILL0_wght400_GRAD0_opsz24.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>192</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>domain_add_24dp_5F6368_FILL0_wght400_GRAD0_opsz24.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddCredor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Add - Credor</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddCentro">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Add - Centro</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>frame2_6</zorder>
   <zorder>frame2</zorder>
   <zorder>label_3</zorder>
   <zorder>frame2_2</zorder>
   <zorder>frame2_3</zorder>
   <zorder>frame2_4</zorder>
   <zorder>frame2_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
